### ISSUE TITLE
Reobf tasks for JarJar tasks are not created

### ISSUE THREAD

**Issue by Jonathing** [MinecraftForge/ForgeGradle#883]
Related to #881. Seems to be an issue with using `configureEach` at plugin application rather than at `project.afterEvaluate`.

![Original Discord Message](https://cdn.moddinglegacy.com/924488901699284142-Discord_hLXh5o4xO5.png)

https://discord.com/channels/313125603924639766/801175194298744902/1009829595803041813

**Comment by Jonathing** [comment]
Closing this as an invalid solution to #881. `configureEach` does in fact work but is negated by how JarJar tasks on creation are disabled by default until the end user in their own `build.gradle` file enables it manually or includes a JarJar dependency. I shall leave it up to our friends @OrionDevelopment and @SizableShrimp 🦐. Here's a screenshot basically summarizing what we talked about in #gradle today.

![Related Discord Message](https://cdn.moddinglegacy.com/102389180778933484-Discord_R1mTEgHEdo.png)

https://discord.com/channels/313125603924639766/801175194298744902/1009874822177751101

Farewell, chat.
